patches/tModLoader/Terraria/Framing.cs.patch

```diff
@@ -8,6 +8,24 @@
  
  namespace Terraria
  {
+@@ -26,13 +_,13 @@
+ 		}
+ 
+ 		private static Point16[][] selfFrame8WayLookup;
+-		private static Point16[][] wallFrameLookup;
++		public static Point16[][] wallFrameLookup;
+ 		private static Point16 frameSize8Way;
+ 		private static Point16 wallFrameSize;
+ 		private static BlockStyle[] blockStyleLookup;
+-		private static int[][] phlebasTileFrameNumberLookup;
++		public static int[][] phlebasTileFrameNumberLookup;
+-		private static int[][] lazureTileFrameNumberLookup;
++		public static int[][] lazureTileFrameNumberLookup;
+-		private static int[][] centerWallFrameLookup;
++		public static int[][] centerWallFrameLookup;
```

Comment:
do these lookups need to be made public?

Reply:
centerWallFrameLookup at least, unless we implement the TileID.Sets above. The others I thought might be useful but don't need to be.